src/Symfony/Component/Validator/Constraints/File.php

```diff
@@ -104,6 +104,8 @@ private function normalizeBinaryFormat($maxSize)
             'ki' => 1 << 10,
             'm' => 1000000,
             'mi' => 1 << 20,
+            'g' => 1000000000,
```

Comment:
i'd rather use constant expression as it is easier to see that the value is correct
```suggestion
            'g' => 1000 * 1000 * 1000,
```

Reply:
Great idea. Could you add the change for 'm' as well?

Reply:
as this is a bugfix it's advisable to leave it for the next commit

if you want to change it (i certainly like code consistency) this is the line:
```php
    'm' => 1000 * 1000,
```